src/fastembed.ts

```diff
@@ -127,7 +127,13 @@ export class FlagEmbedding extends Embedding {
 
     const tokenizer = this.loadTokenizer(modelDir, maxLength);
 
-    const modelPath = path.join(modelDir.toString(), "model_optimized.onnx");
+    const modelPath = path.join(
+      modelDir.toString(),
+      model === EmbeddingModel.MLE5Large ||
+        model === EmbeddingModel.AllMiniLML6V2
+        ? "model.onnx"
+        : "model_optimized.onnx",
+    );
```

Comment:
I have to admit that this is not a very clean implementation, but it's still simple enough.